### ISSUE TITLE
Github caused crash

### ISSUE THREAD

**Issue by HarishTeens** [codesandbox/codesandbox-client#2629]
<h1>💥 Crash Report</h1>

  <h2>I pressed github , and then the app crashed</h2>

  >I just created a new React sandbox and then under configuration i pressed github , then the app crashed

  <h3>sandbox link:https://new.csb.app/</h3>

  <h3>Crash Details</h3>

  <details>
  <summary>Environment</summary>

  | Browser |  Version  | Operating System |
  | ------- | --------- | ---------------- |
  | chrome | 77.0.3865 | Windows 10           |

  **Route:**
  https://codesandbox.io/s/new

  </details>

  <details>
  <summary>Error Message</summary>

  ```bash
  TypeError: Cannot read property 'integrations' of null
  TypeError: Cannot read property 'integrations' of null
    at github (https://codesandbox.io/static/js/app.e345c11d.js:438:407010)
    at Li (https://codesandbox.io/static/js/common.19be7978.chunk.js:1:2052774)
    at Ka (https://codesandbox.io/static/js/common.19be7978.chunk.js:1:2095866)
    at Fa (https://codesandbox.io/static/js/common.19be7978.chunk.js:1:2081589)
    at Na (https://codesandbox.io/static/js/common.19be7978.chunk.js:1:2078609)
    at Ra (https://codesandbox.io/static/js/common.19be7978.chunk.js:1:2077169)
    at https://codesandbox.io/static/js/common.19be7978.chunk.js:1:2039070
    at https://codesandbox.io/static/js/common.19be7978.chunk.js:1:2113916
    at co (https://codesandbox.io/static/js/common.19be7978.chunk.js:1:2038799)
    at po (https://codesandbox.io/static/js/common.19be7978.chunk.js:1:2039016)
    at ho (https://codesandbox.io/static/js/common.19be7978.chunk.js:1:2038951)
    at Ia (https://codesandbox.io/static/js/common.19be7978.chunk.js:1:2077596)
    at https://codesandbox.io/static/js/common.19be7978.chunk.js:1:2028438
    at HTMLDocument.r (https://codesandbox.io/static/js/common.19be7978.chunk.js:1:122867)
  
    in github
    in div
    in div
    in t
    in elements__Container
    in jd
    in div
    in t
    in div
    in t
    in div
    in div
    in t
    in styled.div
    in div
    in t
    in elements__Container
    in t
    in n
    in r
    in n
    in r
    in t
    in t
    in div
    in t
    in elements__Content
    in n
    in t
    in withRouter(n)
    in div
    in t
    in elements__Container
    in Unknown
    in r
    in t
    in withRouter(r)
    in DragDropContext(withRouter(r))
    in t
    in t
    in l
    in t
  ```

  </details>

**Comment by lbogdan** [comment]
This looks like a duplicate of #2674 , which was fixed in #2676 . @HarishTeens Can you please confirm it's fixed for you?

**Comment by HarishTeens** [comment]
Yes it is . Thanks @lbogdan
